assets/css/_windowTitleBar.css

```
@@ -218,6 +218,8 @@ html.desktop-active .fsbl-icon-highlighted {
 
 .linker-group-accessible {
     width: 1em !important;
+    padding-bottom: 1px;
+    margin-bottom: 4px;
 }
```

The white on the light colours are difficult to see while updating for accessibility could we adjust the contrast between the text and background by using the following, what are your views on this @joel-ciq ?
```suggestion
   text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.79);
```